style.css

```diff
@@ -554,12 +596,43 @@ hr#last {
 }
 
 /* For Smarth phones */
-@media screen and (min-width: 280px) and (max-width: 768px) {
+@media screen and (min-width: 270px) and (max-width: 768px) {
   html {
     font-size: 50%;
   }
 
-  #burger-menu {
+  .project-first {
+    display: grid;
```

Comment:
- [ ] Since you have been building your portifolio in mobile first version, you need to make sure that you ensure that you follow the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#project-requirements) to implement desktop version. According to the project requirements you should ensure to use media queries in your CSS file to make your desktop version look like the template you selected (Desktop: >=768px). Therefore I suggest to use media query that matches the requirement description ☺️.

Your designs at  790px
![image](https://github.com/ZabihullahNooriWardak/Zabih-portfolio/assets/99394374/6ae66e4d-3f09-4f15-b128-b4c4a4c5e053)

![image](https://github.com/ZabihullahNooriWardak/Zabih-portfolio/assets/99394374/84303709-0b85-4f86-9472-cbc96bfb9c5f)

Original design 😎 
![image](https://github.com/ZabihullahNooriWardak/Zabih-portfolio/assets/99394374/e81c5e87-4a75-47b9-ae8b-3c9d94cc731a)